Exercise_2/main.cpp

```diff
@@ -0,0 +1,179 @@
+#include <string>
+#include <locale>
+#include <iostream>
+#include <fstream>
+#include <algorithm>
+#include <vector>
+#include <limits>
+#include <climits>
+#include <list>
+
+using namespace std;
+
+struct nodes {
+    int start_node;
+    int end_node;
+    int distance;
+};
+
+struct table {
+    int vertex;
+    int distance;
+    int previos_vector;
+};
+
+
+vector<int> create_set_of_values_from_file(const vector<nodes> &all_data){
```

Comment:
If you want a set of values, use the `std::set` (: Or event better `std::unordered_set`